src/pages/About/index.js

```diff
@@ -1,5 +1,36 @@
-import { Wrapper } from "./style";
-
+import { Card, CardsContainer, HighlightText, Wrapper } from "./style";
+import ComunnityImg from '../../assets/community.svg';
+import EmpowermentImg from '../../assets/empowerment.svg';
+import LightBoxImg from '../../assets/light-box.svg';
 export const About = () => {
-  return <Wrapper>About</Wrapper>;
+  return (
+    <Wrapper>
+          <h2>SOBRE NÓS</h2>
+          <p>Somos uma comunidade voltada a incentivar a cultura open source no curso</p>
```

Comment:
Acho uma boa resumir as duas tags de parágrafo em uma, mesmo que ele não fique exatamente igual ao design do figma

```suggestion
          <p>Somos uma comunidade voltada a incentivar a cultura open source no curso de Ciencia da computação</p>
```